src/components/TransactionBlock/AssetSwapTransactionBlock.tsx

```diff
@@ -164,8 +199,24 @@ const AssetSwapTransactionBlock = ({
         const offers = await updateAvailableOffers();
         if (!active || !offers) return;
 
+        const usdValuesGas = await Promise.all(offers.map((offer) => getGasSwapUsdValue(offer)));
+        const valuesToReceiveRaw = offers.map((offer) => {
+          const toAsset = availableToAssets
+            ? availableToAssets?.find((availableAsset) =>
+                addressesEqual(availableAsset.address, selectedToAsset?.address)
+              )
+            : null;
+
+          return targetAssetPriceUsd
+            ? +ethers.utils.formatUnits(offer.receiveAmount, toAsset?.decimals) * targetAssetPriceUsd
+            : +ethers.utils.formatUnits(offer.receiveAmount, toAsset?.decimals);
```

Comment:
can we simplify this because it occupies 2 lines of same lenght code just because in once case it's multiplied by another digit ```suggestion
          return targetAssetPriceUsd
            ? +ethers.utils.formatUnits(offer.receiveAmount, toAsset?.decimals) * targetAssetPriceUsd
            : +ethers.utils.formatUnits(offer.receiveAmount, toAsset?.decimals);
```

Reply:
easiest way this can be just `* (targetAssetPriceUsd ?? 1)`